pkgs/by-name/ti/tiramisu/package.nix

```
@@ -7,16 +7,14 @@
   vala,
 }:
 
-stdenv.mkDerivation rec {
-  pname = "tiramisu";
-  # FIXME: once a newer release in upstream is available
-  version = "2.0-unstable-2023-03-29";
+stdenv.mkDerivation (finalAttrs: {
+  name = "tiramisu";
```

I'm afraid this is incorrect. According to [the docs](https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/stdenv.chapter.md#using-stdenv-sec-using-stdenv):

> Specifying a `name` and a `src` is the absolute minimum Nix requires. For convenience, you can also use `pname` and `version` attributes and `mkDerivation` will automatically set `name` to `"${pname}-${version}"` by default.

```suggestion
  pname = "tiramisu";
```

If we were to use `name`, we'd have to set it to `name = "tiramisu-2.0.20240610";`, but the preferred way is to have this separately stored as `pname` and `version`.